packages/nestjs-auth-google/src/exceptions/auth-google-missing-email.exception.ts

```diff
@@ -0,0 +1,10 @@
+import { RuntimeException } from '@concepta/nestjs-exception';
+
+export class AuthGoogleMissingEmailException extends RuntimeException {
+  constructor(
+    message = 'Google did not return an email address for the user.',
+  ) {
+    super(message);
```

Comment:
This will default to 500... should add a friendly error message. I can't remember if it does this by default.